ui/design-system-core/src/components/Spinner/index.tsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import { tw } from '@twind/core';
 
-export interface ISpinnerProps {
+import { Color } from '../types/common.types';
+
+export type SpinnerProps = {
   size?: 'sm' | 'md' | 'lg' | 'xl' | 'xxl';
-  color?: string;
-}
+  color?: string | Color;
```

Comment:
Using string will lead a dev to a use a color that's not defined or known by tailwind or twind. That's why we are explicitly defining colors we are using on Colors type.